### DISCUSSION TITLE
steering_committee: Tim Swast

### DISCUSSION THREAD

**Discussion by cpcloud** [ibis-project/governance#3] [Steering committee member] [answered]
I would like to nominate Tim Swast (@tswast) for steering council membership.

Tim has been a consistent positive influence on and contributor to the project for many years and has shown consistent commitment to maintaining the health of the project.

Tim has primarily contributed to the BigQuery backend and has helped with numerous improvements, features and design with that backend as well.

I think Tim would be a great addition to the steering council.

**Comment by gforsyth** [comment]
I am definitely in favor of this.  +1 from me.

**Comment by tswast** [comment]
Thanks for considering me. I'd love this. It'd be great to work closer with you all.

**Comment by jcrist** [comment]
+1 from me as well

**Comment by cpcloud** [comment, marked as answer]
+1, with three votes, the motion carries!

Welcome to the steering committee @tswast, great to have you!
